launcher/spec.go

```diff
@@ -73,9 +73,14 @@ func (s spec[T]) NewService(db persistence.Storage) (svc service.Service, grpcOp
 	return s.newService(db)
 }
 
+func (s spec[T]) HasStorageFunction() bool {
```

Comment:
I am not 100 % sure I understand why this is necessary but it seems to work :D Can we make this unexported -> `hasStorageFunction`